libc/test/src/__support/CMakeLists.txt

```diff
@@ -181,4 +181,12 @@ add_subdirectory(File)
 add_subdirectory(RPC)
 add_subdirectory(OSUtil)
 add_subdirectory(FPUtil)
-add_subdirectory(HashTable)
+
```

Comment:
The tests are build after all the sources right? It should be easier to just check the target like `if(TARGET libc.src.<whatever>.hsearch)`